dotCMS/src/main/java/com/dotmarketing/cms/factories/PublicCompanyFactory.java

```diff
@@ -118,19 +116,23 @@ public static void createDefaultCompany() {
 			CompanyUtil.update(c);
 
 			/* Set the DM logo */
-			File f = new File(FileUtil.getRealPath("/html/images/shim.gif"));
+			BufferedInputStream in = null;
+			final ByteArrayOutputStream baout = new ByteArrayOutputStream();
+			final File file = new File(FileUtil.getRealPath("/html/images/shim.gif"));
 
-			BufferedInputStream in = new BufferedInputStream(new FileInputStream(f));
+			try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)